File: test/c++/src/company.cpp
#include <company.hpp>
#include <iostream>

Company::Company()
	{
	}
	
Company::~Company()
	{
	}
	
void Company::addEmployee(Employee emp)
	{
	m_employees.push_back(emp);
	}
	
void Company::printEmployees()
	{
	for (unsigned int I = 0; I < m_employees.size(); I++)
		std::cout << m_employees[I].getName() << std::endl;
	}


	

File: test/java/src/Company.java
import java.util.*;


public class Company
	{
	private Map m_employees;
	
	public Company()
		{
		m_employees = new HashMap();
		}
		
	public void addEmployee(Employee emp)
		{
		m_employees.put(emp.getId(), emp);
		}
		
	public void printEmployees( )
		{
		Iterator it = m_employees.values().iterator();
		Employee emp;
		
		while (it.hasNext())
			{
			emp = (Employee)it.next();
			System.out.println(emp.getName());
			}
		
		}
	}
